source/_integrations/broadlink.markdown

```diff
@@ -34,10 +37,16 @@ The entities have the same name as the device by default. To change the name, ic
 
 The entities are divided into three subdomains:
 
+- [Climate](#climate)
 - [Remote](#remote)
 - [Sensor](#sensor)
 - [Switch](#switch)
 
+## Climate
+
+The `climate` entities allow you to monitor and control Broadlink thermostats. For now only the Floureon HY03WE is supported and tested. Others may work, if not please open an issue.
```

Comment:
```suggestion
The `climate` entities allow you to monitor and control Broadlink thermostats. For now, only the Floureon HY03WE is supported and tested. Others may work.
```

Adding support for new devices is a feature request. Feature requests go into the forums, not the issue tracker 😉

Reply:
The supported models are on top. I think we should keep only this part:

The `climate` entities allow you to monitor and control Broadlink thermostats.